tests/foreman/cli/test_computeresource_rhev.py

```diff
@@ -0,0 +1,382 @@
+# -*- encoding: utf-8 -*-
+"""Usage::
+
+    hammer compute_resource [OPTIONS] SUBCOMMAND [ARG] ...
```

Comment:
that data is already present in `computeresource.py`, so please remove it here and move `:Requirement: Computeresource` into top of the file